ush/templates/FV3LAM_wflow.xml

```diff
@@ -489,6 +594,8 @@ MODULES_RUN_TASK_FP script.
 ************************************************************************
 ************************************************************************
 -->
+
+<!--
```

Comment:
These will need to be remove to deliver NCL graphics to the web. Please also address the commented tasks below.

Reply:
uncommented "ARCHIVE_TN" and "RUN_NCL_ZIP_TN".